### DISCUSSION TITLE
Example of building a table with bootstrap (4or5) that defines what "data" looks like?

### DISCUSSION THREAD

**Discussion by steelliberty** [helloflask/bootstrap-flask#315] [Q&A] [answered]
I have not been able to find an example using render_table(data) that provides a description or even a look at the data structure. 

Does anyone have a reference to "building a table with bootstrap-flask? 

Thank you, 

JM

**Comment by PanderMusubi** [comment]
Do you mean https://github.com/helloflask/bootstrap-flask/blob/master/examples/bootstrap5/templates/table.html#L47 ?

An example is Table without SQLAlchemy at the bottom of http://109.205.179.12/table

**Reply by steelliberty** [reply]
Hi, and thanks - yes, I saw that -- however when I created a simple table from data =[{}...] and titles =[t1,t2,t3] I got a single letter as the column titles -- a letter from each title and nothing at all in the data portion of the data. 

data = [{'feedname': 'morning news', 'feedlink': 'https://rss.app/feeds/ydydfyasss.xml', 'author_name': 'Greaseman'}, {}..]
titles = ['feedname', 'feedlink', 'author_name']

I am not using and sqlAlchemy data is generated from an api --

thanks again,

**Comment by PanderMusubi** [comment]
Then probably the string of the title gets indexed. Try the example and see exactly what is used there as data. Does that help?

**Reply by steelliberty** [reply, marked as answer]
Hi -- I saw that Titles requires list of tuples [('title','alt-title'),()...] Once i figured that out it worked. 

Thank you for the help.
